documentation/modules/rn-2.5.adoc

```diff
@@ -74,16 +74,19 @@ In addition to standard password authentication, the following authentication me
 
 The validation service includes default validation rules for virtual machines from OpenStack. link:https://issues.redhat.com/browse/MTV-508[(MTV-508)]
 
-
 .VDDK is now optional for VMware vSphere providers
 
 The VMware vSphere source provider can now be created without specifying a VDDK `init` image. It is strongly recommended to create a VDDK `init` image to accelerate migrations.
 
 .Deployment on OKE enabled
 
-// adding the link to the "About OKE" docs as it is better than nothing
 In {project-short} 2.5.3, deployment on OpenShift Kubernetes Engine (OKE) has been enabled. For more information, see link:https://docs.openshift.com/container-platform/4.14/welcome/oke_about.html[About OpenShift Kubernetes Engine]. link:https://issues.redhat.com/browse/MTV-803[(MTV-803)]
 
+.Migration of VMs from {rhv-short} to destination storage classes with encrypted RBD now supported
+
+In {project-short} 2.5.4, migration of VMs from {rhv-full} ({rhv-short}) to destination storage classes that have encrypted RADOS Block Devices (RBD) destination storage classes is now supported.
+
+To make use of this new feature, set the value of the parameter `controller_block_overhead` to `2Gi`, following the procedure in link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.5/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#configuring-mtv-operator_mtv[Configuring the MTV Operator]. link:https://issues.redhat.com/browse/MTV-851[(MTV-851)]
```

Comment:
Please can I ask why a `LINK` and not an `XREF`?

Reply:
@ahadas @liranr23 We're leaving this as a link for now -- it turns out creating a xref between documents in the repo is a bigger challenge than one might think. Using a link will work until 2.6 goes GA, by then, we expect to solve the problem.